docs/spec/loops.md

```diff
@@ -127,20 +127,22 @@ resource vnet 'Microsoft.Network/virtualNetworks@2018-11-01' = {
 ```
 
 ### Nested loops and filtering.
-The example below demonstrates a nested loop combined with filters at each loop. Filters must be expressions that evaluate to a boolean value.
+The example below demonstrates a nested loop combined with a filtered resource loop. Filters must be expressions that evaluate to a boolean value.
 
 ```bicep
-resource parentResources 'Microsoft.Example/examples@2020-06-06' = [for parent in parents where parent.enabled: {
+resource parentResources 'Microsoft.Example/examples@2020-06-06' = [for parent in parents: if(parent.enabled) {
```

Comment:
>if [](start = 91, length = 2)

I reordered some of the tokens in the syntax (explanation in the description). If there are concerns or objections, we can put it on the discussion meeting agenda.